engineering-team.html

```diff
@@ -31,7 +31,7 @@
         <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
     </head>
 
-<body style="font-family: 'Montserrat', sans-serif; color: black">
```

Comment:
Removed the black font color of the engineering team page. Is it ok?

Reply:
yeah